### ISSUE TITLE
Improve the Footer Design and Accessibility

### ISSUE THREAD

**Issue by diyakotru** [RAJIV81205/Weather-Info#102]
Description:
The current footer feels unbalanced and lacks modern styling and accessibility. Social icons have no labels or hover effects, and content alignment is uneven.

Suggestions:
Use Flexbox/Grid for better layout.
Add hover effects and aria-labels to icons.
Enhance visual styling with better padding, colors, and font hierarchy.

Screenshot : 

<img width="1895" height="198" alt="Image" src="https://github.com/user-attachments/assets/38557768-f9c7-4536-8172-e4a6b780321a" />

@RAJIV81205 I would like to work on this issue kindly assign it to me

**Comment by RAJIV81205** [comment]
@diyakotru assigned

**Comment by diyakotru** [comment]
@RAJIV81205 hlo i have raised the pr kindly review it .
